adapter/config/tpr/config.go

```diff
@@ -45,6 +46,14 @@ func (e *Config) GetObjectMeta() meta_v1.Object {
 	return &e.Metadata
 }
 
+// DeepCopyObject implements a mandatory interface
+func (e *Config) DeepCopyObject() runtime.Object {
+	if e == nil {
+		return nil
+	}
+	return e
```

Comment:
IMHO DeepCopyObject for something that just directly returns the reference might be misleading. Perhaps we should call it Reference()?